.travis.yml

```diff
@@ -3,7 +3,6 @@ dist: xenial
 python:
   - 3.8
 env:
-  - TOXENV=django22-celery44
```

Comment:
Should we hold off on removing this testing until edx-platform has upgraded to celery 5.x?